lib/trino-orc/src/main/java/io/trino/orc/OrcWriter.java

```diff
@@ -88,13 +88,13 @@
 {
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(OrcWriter.class).instanceSize();
 
-    private static final String PRESTO_ORC_WRITER_VERSION_METADATA_KEY = "presto.writer.version";
```

Comment:
How it's being used?

Reply:
i don't think it is used in any way within the project. We write this so that we can distinguish files written by different Trino versions later on, but i don't think we had to, at least not recently.